backend/src/helpers/index.ts

```diff
@@ -0,0 +1,17 @@
+import { paramCase, Options } from 'change-case';
```

Comment:
each helper should be in separate folder and file
move this helper to the shared folder